src/utils/validation/registration/referenceValidation.ts

```diff
@@ -44,9 +55,21 @@ export const baseReference = Yup.object().shape({
 const journalPublicationInstance = Yup.object().shape({
   type: Yup.string().oneOf(Object.values(JournalType)).required(ErrorMessage.REQUIRED),
   peerReviewed: peerReviewedField,
-  articleNumber: Yup.number().typeError(ErrorMessage.INVALID_FORMAT).min(0, ErrorMessage.MUST_BE_POSITIVE),
-  volume: Yup.number().typeError(ErrorMessage.INVALID_FORMAT).min(0, ErrorMessage.MUST_BE_POSITIVE),
-  issue: Yup.number().typeError(ErrorMessage.INVALID_FORMAT).min(0, ErrorMessage.MUST_BE_POSITIVE),
+  articleNumber: Yup.number()
+    .typeError(ErrorMessage.INVALID_FORMAT)
+    .min(0, ErrorMessage.MUST_BE_POSITIVE)
+    .transform(emptyStringToNull)
+    .nullable(),
+  volume: Yup.number()
+    .typeError(ErrorMessage.INVALID_FORMAT)
+    .min(0, ErrorMessage.MUST_BE_POSITIVE)
+    .transform(emptyStringToNull)
+    .nullable(),
+  issue: Yup.number()
+    .typeError(ErrorMessage.INVALID_FORMAT)
+    .min(0, ErrorMessage.MUST_BE_POSITIVE)
+    .transform(emptyStringToNull)
+    .nullable(),
```

Comment:
synes egentlig Yup burde fikset dette også da. Lett å lage PR til Yup? ser det er rapportert som bug der også: https://github.com/jquense/yup/issues/1290

Reply:
Enig, du dukket faktisk opp med en link i en annen populær tråd om dette,  fra når du fiksa det samme for måned og dag 😅
så ikke bare vi som syns dette er tungvint